app/components/track-page/course-card-list.ts

```diff
@@ -36,6 +36,38 @@ export default class TrackPageCourseCardList extends Component<Signature> {
         course: course,
       };
     });
+
+    return coursesWithProgress.sort((a, b) => {
+      const aRepo = a.repositoryWithMostProgress;
```

Comment:
```suggestion
      const aRepository = a.repositoryWithMostProgress;
```

(Don't abbreviate unless it significantly improves readability - applies to other variable names here too)